### DISCUSSION TITLE
When inserting 1 million records, an error occurred and BE crashed directly

### DISCUSSION THREAD

**Discussion by xxtssssss** [apache/doris#29844] [A - General / Q&A]
报错如下 
![7HF13B){J@926%UW}PBF`IH](https://github.com/apache/doris/assets/117354208/9ddd0d18-6ffd-423d-9138-26b3927cf6e5)

LF4J: Actual binding is of type [org.slf4j.impl.Reload4jLoggerFactory]
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
*** Query id: 0-0 ***
*** Aborted at 1704439601 (unix time) try "date -d @1704439601" if you are using GNU date ***
*** Current BE git commitID: Unknown ***
*** SIGABRT unkown detail explain (@0x71a0c) received by PID 465420 (TID 0xfff2a9d77bc0) from PID 465420; stack trace: ***

**Comment by xxtssssss** [comment]
是sql的问题吧数据量也不大就200M

**Reply by LemonLiTree** [reply]
具体说说导入操作？包括版本

**Reply by xxtssssss** [reply]
导入就是kafka消费进去的，感觉就是消费到某个量或者某张表就挂了 经常出现 数据量也不大就几百M，版本是1.2.5

**Reply by LemonLiTree** [reply]
routineload？测试环境还是生产？

**Reply by xxtssssss** [reply]
生产环境 我在我本地跑过一次 没问题 部署环境是arm64的国产麒麟系统 一样的版本和配置   kafka消费部分代码
![414T_%1UZ${H)0 9H`39V{G](https://github.com/apache/doris/assets/117354208/0c5412e5-d58e-467e-bbfe-36794f2aec26)

**Reply by xxtssssss** [reply]
服务器配置16核 64G运行内存

**Reply by LemonLiTree** [reply]
你这是一个jsondata 执行一次streamload？

**Reply by xxtssssss** [reply]
是的

**Reply by xxtssssss** [reply]
是不是有问题呢 但是这套代码在其他地方用并没有出现这种情况呢

**Reply by LemonLiTree** [reply]
先赞批到文件，然后 30s或者1分钟 提交一次streamload。

**Reply by xxtssssss** [reply]
我增量数据很少的 ，能修改某个配置来解决吗 这套代码在其他生产环境数据量级比我现在这个数据量大多了

**Reply by xxtssssss** [reply]
![image](https://github.com/apache/doris/assets/117354208/049bfd31-4bb8-404c-a0bc-544189a8880b)
这个问题

**Reply by LemonLiTree** [reply]
这个是正常的日志
